tripper/backends/sparqlwrapper.py

```diff
@@ -123,14 +123,39 @@ def _get_sparql_query_type(self, query: str) -> str:
         # either the beginning of the string or following a newline
         # or a period. It then matches one of the keywords.
         pattern = re.compile(
-            r"(?:(?<=^)|(?<=[\.\n]))\s*(ASK|CONSTRUCT|SELECT|DESCRIBE)\b",
+            r"(?:(?<=^)|(?<=[\.\n]))\s*(ASK|CONSTRUCT|SELECT|DESCRIBE|"
+            r"DELETE|INSERT)\b",
             re.IGNORECASE,
         )
         match = pattern.search(query)
         if match:
             return match.group(1).upper()
         return "UNKNOWN"
 
+    def update(
+        self,
+        update_object: str,
+        **kwargs,  # pylint: disable=unused-argument
+    ) -> None:
+        """Update triplestore with SPARQL quary.
+
+        Arguments:
+            query_object: String with the SPARQL query.
+            kwargs: Additional backend-specific keyword arguments.
+
+        Note:
+            This method is intended for INSERT and DELETE queries.  Use
+            the query() method for ASK/CONSTRUCT/SELECT/DESCRIBE queries.
+        """
+        query_type = self._get_sparql_query_type(update_object)
+        if query_type not in ("DELETE", "INSERT"):
```

Comment:
If you put this in, I think you should also add a test to it. It is quite quick, there is already one that can be copied using query.

Reply:
Added some tests for these as well